configure.ac

```diff
@@ -568,6 +568,19 @@ if test x$enable_music_midi_timidity != xyes -a \
     AC_MSG_WARN([MIDI support disabled])
 fi
 
+AC_ARG_ENABLE([music-gme],
```

Comment:
As it is, the configury is broken: here is a patch to fix it: [patch_378b.txt](https://github.com/libsdl-org/SDL_mixer/files/8605149/patch_378b.txt)

And after that, the PR definitely needs squashing.